analysis/SConstruct

```diff
@@ -18,12 +18,13 @@ gs.log.start_log(mode, vers)
 env = Environment(ENV = {'PATH': os.environ['PATH']}, 
                   IMPLICIT_COMMAND_DEPENDENCIES = 0,
                   BUILDERS = {
-                              # 'BuildMatlab': Builder(action = gs.build_matlab),
+                              'BuildMatlab': Builder(action = gs.build_matlab),
```

Comment:
why uncomment?

Reply:
Was testing. Will recomment.

Reply:
Whoops, forgot to do this before merging. Done here though https://github.com/gslab-econ/template/commit/e5659df655beab7ca7db3fb39336873814414d49.